akka-http-core/src/main/scala/akka/http/scaladsl/model/Uri.scala

```diff
@@ -278,8 +321,11 @@ object Uri {
    * Parses the given string as if it were the value of an HTTP/2 ":path" pseudo-header.
    * The result is a path and a query string as defined in
    * https://tools.ietf.org/html/rfc7540#section-8.1.2.3
-   * If strict is `false`, accepts unencoded visible 7-bit ASCII characters in addition to the RFC.
    * If the given string is not a valid path or query string the method throws an `IllegalUriException`.
+   *
+   * @param mode if `Relaxed`, accepts unencoded visible 7-bit ASCII characters in addition to the RFC.
+   * @return path and percent-encoded query string. When in in 'relaxed' mode, characters not permitted by https://tools.ietf.org/html/rfc3986#section-3.4
+   *         *  are already automatically percent-encoded here
```

Comment:
```suggestion
   * are already automatically percent-encoded here
```

Reply:
@ennru  I think it should be:

```suggestion
   *         are already automatically percent-encoded here
```

To align with the previous line.